src/app/screens/sendStx/steps/Step1SelectRecipient.tsx

```diff
@@ -0,0 +1,146 @@
+import InputScreen from '@components/inputScreen';
+import { useBnsName, useBnsResolver } from '@hooks/queries/useBnsName';
+import useDebounce from '@hooks/useDebounce';
+// import { useBnsResolver } from '@hooks/queries/useBnsName';
+// import useDebounce from '@hooks/useDebounce';
+import useWalletSelector from '@hooks/useWalletSelector';
+import { validateStacksAddress } from '@stacks/transactions';
+import Button from '@ui-library/button';
+import Input from '@ui-library/input';
+import { FeedbackVariant } from '@ui-library/inputFeedback';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+type StxRecipientScreenProps = {
+  recipientAddress: string;
+  setRecipientAddress: (address: string) => void;
+  // recipientDomain: string;
+  setRecipientDomain: (address: string) => void;
+  memo: string;
+  setMemo: (memo: string) => void;
+  onNext: () => void;
+  isLoading: boolean;
+  header?: React.ReactNode;
+};
+
+const MemoInput = styled(Input)`
+  input {
+    height: 64px;
+    padding-top: 0px;
+  }
+`;
+
+const RecipientInput = styled(Input)`
+  min-height: 125px;
+`;
+
+interface InputFeedback {
+  variant: FeedbackVariant;
+  message: string;
+}
+
+function Step1SelectRecipient({
```

Comment:
Do we need to have StepX as a prefix? 😅 If we reorder the steps or add a step in between, this naming will be a chore to redo. Maybe better to remove the prefixes?

Reply:
The StepX prefix helped me a lot when troubleshooting / debugging, it's easier to go to the right file. I think it's worth the future chore of renaming it if we reorder the steps / add a step in-between. In my opinion, renaming is very trivial.